plugins/grid-bookmark/src/GridBookmarkWidget/components/BookmarkGrid.tsx

```diff
@@ -81,16 +79,17 @@ const BookmarkGrid = observer(function ({
           {
             field: 'locString',
             headerName: 'Bookmark link',
-            width:
-              bookmarkRows.length > 0
-                ? measureGridWidth(bookmarkRows.map(row => row.locString))
-                : measureText('Bookmark link'),
+            width: Math.max(
```

Comment:
not very important but i think this is sort of an expected outcome: we just want the maximum of these two states